stock-filters/ChirpVillageBuilder.py

```diff
@@ -20,14 +22,41 @@
 
 
 def perform(level, box, options):
-    yard_generator = YardGenerator(level, box)
+    print("RUNNING CHIRP VILLAGE GENERATOR!! ")
+    # Yard Generation
+    print("Surface")
+    surface = Surface(level, box)
+    print("Yard Generator")
+    yard_generator = YardGenerator(level, box, surface)
+    print("generate yards")
     yard_generator.generate_yards()
+    print("getting Surface")
     surface = yard_generator.surface
-    BlockUtils.calculate_biomes_on_surface(level, surface)
+
+    surface.visualize_yards()
+    surface.visualize_heights()
+    surface.visualize_steepness()
+
+    # Biome Adaptibility
+    # BlockUtils.calculate_biomes_on_surface(level, surface)
```

Comment:
:(

Reply:
RIP that was commented out form the live session...

Reply:
Can it be re-added? Unless its breaking something?